content/en/about/faq/traffic-management/api.md

```diff
@@ -0,0 +1,10 @@
+---
+title: Which API version should I use?
+weight: 10
+---
+
+Always use the primary/latest API version declared in the CRD, as this will give you the most up-to-date functionality and the system will handle any necessary version conversions transparently.
```

Comment:
This is not quite true, any version gives up-to-date functionality. 

I think we want to convey a message that it _it really doesn't matter_. There is no reason not to use the latest (unless you are an integration author, then see below), but there is also no strong need to spending time migrating existing resources or anything like that

Reply:
Hey @howardjohn, is this [link](https://istio.io/latest/docs/ops/integrations/) sufficient for displaying info on integration author in the FAQ